cmd/lncli/commands.go

```diff
@@ -2310,6 +2310,15 @@ var listInvoicesCommand = cli.Command{
 	Name:     "listinvoices",
 	Category: "Payments",
 	Usage:    "List all invoices currently stored.",
+	Description: `
+	This command enables the retrieval of all invoices currently stored
+	within the database. It has full support for pagination-style responses,
+	allowing users to query for specific invoices through their add_index.
+	This can be done by using either the first_index_offset or
+	last_index_offset fields included in the response. The reversed flag is
+	set by default in order to paginate backwards. If you wish to paginate
+	forwards, you must explicitly set the flag to false. If none of the
+	parameters are specified, then the last 100 invoices will be returned.`,
```

Comment:
Usage description of "index_offset" need an update?

Reply:
Will update it in an upcoming commit.